src/index.ts

```diff
@@ -6,3 +8,5 @@ export { Component, Selector } from './selector';
 export * as Transformations from './transformations';
 
 export { Renderer } from './Renderer';
+
+window.ResizeObserver = window.ResizeObserver ?? ResizeObserver;
```

Comment:
> another thing is weird. Unit test doesn't pass event if I revert my changes...

Hm... That's because the build is tested in `node` environment, where there is no `window` defined:

> ReferenceError: window is not defined

:thinking:

Reply:
To work it around, I can suggest using the [global](https://www.npmjs.com/package/global) npm package:

```ts
import window from 'global';
import ResizeObserver from 'resize-observer-polyfill';

window.ResizeObserver = window.ResizeObserver ?? ResizeObserver;
```

Reply:
So importing like that:
```
import window from 'global';
```
 doesn't work:
```
TS7016: Could not find a declaration file for module 'global'. '/Users/prezly/Documents/code/content-renderer-react-js/node_modules/global/window.js' implicitly has an 'any' type.   Try `npm i --save-dev @types/global` if it exists or add a new declaration (.d.ts) file containing `declare module 'global';`
```
So I've tried to use `require` but `Can't find variable: require `